### PR TITLE
[DC-1341] Refactor generate_unioned_ehr_dataset.sh shell to python script

### PR DESCRIPTION
**Pull request by ksdkalluri**

### CODE REVIEW

File: data_steward/tools/generate_unioned_ehr_dataset.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+    impersonation_creds = auth.get_impersonation_credentials(
+        args.run_as_email, CDR_SCOPES)
+
+    client = BigQueryClient(args.project_id, credentials=impersonation_creds)
+
+    unioned_ehr_backup = create_dataset(client, args.release_tag, 'backup')
+
+    LOGGER.info('Creating destination CDM tables...')
+    create_cdm_tables(client, unioned_ehr_backup)
```
Comment (hiro-mishima) [inline data_steward/tools/generate_unioned_ehr_dataset.py:232]: This does not cover `aou_death`, since it's not a part of `CDM_TABLES`.

**Review by hiro-mishima — COMMENTED** [review]
Hi @ksdkalluri, I know you are in the middle of updating the script, but please let me add some comments from my initial review.

**Review by hiro-mishima — CHANGES_REQUESTED** [review]
Thanks for the updates, @ksdkalluri! I added some comments. All are minor. Thanks!

File: data_steward/tools/generate_unioned_ehr_dataset.py (hunk 1 above)
Comment (ksdkalluri) [reply data_steward/tools/generate_unioned_ehr_dataset.py:140]: I think we still need this.

**Review by hiro-mishima — CHANGES_REQUESTED** [review]
Thanks for updating the scripts, Krishna! I added a couple of comments.

File: data_steward/tools/pipeline_utils.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    """
+    Create all CDM tables in the specified dataset.
+
+    :param client: BigQueryClient
+    :param dataset: Target dataset name
+    :return: None
+
+    Note: Recreates any existing tables
+    """
+    for table in CDM_TABLES + [AOU_DEATH]:
```
[the remark this answers is not in the document]
Comment (ksdkalluri) [reply data_steward/tools/pipeline_utils.py:127]: I can just remove the redundant table creation and it should be good right?

**Review by hiro-mishima — APPROVED** [review]
Thanks for all the changes! All look good!!

**Pull request merged**